lib/task/import/csvRepositoryImportTask.class.php

```diff
@@ -176,20 +176,30 @@ public function execute($arguments = [], $options = [])
                 }
 
                 // Handle description status
-                $self->object->descStatusId = $self->translateNameToTermId(
-                    'description status',
-                    $self->rowStatusVars['descriptionStatus'],
-                    [],
-                    $self->status['descriptionStatusTypes'][$self->columnValue('culture')]
-                );
+                if (
```

Comment:
Rather than adding this conditional around all calls to `translateNameToTermId()` I think it would be better to do the null handling inside the `translateNameToTermId()` method to prevent errors like this in the future.